### ISSUE TITLE
@angular-eslint/schematics/dist/convert-tslint-to-eslint/schema.json does not exist

### ISSUE THREAD

**Issue by mpalourdio** [angular-eslint/angular-eslint#1364]
**Description and reproduction of the issue**

Make a fresh new angular 16 project : `ng new testng16`
Add @angular-eslint : `ng add @angular-eslint/schematics`
Try to generate a lib : `ng g lib mylib`

Error : 
`An unhandled exception occurred: Path "/what/ever/path/node_modules/@angular-eslint/schematics/dist/convert-tslint-to-eslint/schema.json" does not exist.
See "/tmp/ng-jKHbaO/angular-errors.log" for further details.`

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@angular-eslint/builder`          | `16.0.0` |
| `ESLint`                           | `8.39.0` |
| `node`                             | `18.14.0` |

```sh
Angular CLI: 16.0.0
Node: 18.14.0
Package Manager: npm 9.3.1
OS: linux x64

Angular: 16.0.0
... animations, cli, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1600.0
@angular-devkit/build-angular   16.0.0
@angular-devkit/core            16.0.0
@angular-devkit/schematics      16.0.0
@schematics/angular             16.0.0
rxjs                            7.8.1
typescript                      5.0.4
```

- [ X] I have updated to the latest supported version of the packages and checked my `ng version` output per the instructions given here.

**Comment by JamesHenry** [comment]
Hi @mpalourdio are you sure this captures everything involved?

I cannot reproduce any issue with the instructions provided. `mylib` is generated just fine

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/900523/236040937-f70a60d0-9526-4df9-88d3-ab8930c6edb7.png">

**Comment by JamesHenry** [comment]
Ok I'm still not sure what caused the error for you, but I can see that file is indeed not present in 16.0.0 - I have released 16.0.1 now to address it, so there shouldn't be any more errors with that

**Comment by mpalourdio** [comment]
Just tested, and I can confirm 16.0.1 fixes the problem.
